core/templates/pages/admin-page/config-tab/admin-config-tab.directive.ts

```diff
@@ -39,11 +39,7 @@ angular.module('oppia').directive('adminConfigTab', [
       controller: [function() {
         var ctrl = this;
         ctrl.isNonemptyObject = function(object) {
-          var hasAtLeastOneElement = false;
-          for (var property in object) {
-            hasAtLeastOneElement = true;
-          }
-          return hasAtLeastOneElement;
+          return !(Object.keys(object).length === 0);
```

Comment:
Should we do it this way: `Object.keys(object).length !== 0`? [For better readability]

Reply:
Done.